types/src/verifier/mod.rs

```diff
@@ -45,6 +53,70 @@ pub trait ProofVerifier {
         }
     }
 
+    fn verify_aggregate_seals(
+        aggregate: &AggregateSealVerifyProofAndInfos,
+    ) -> Result<(), Box<dyn StdError>> {
+        if aggregate.infos.is_empty() {
+            return Err("no seal verify infos".into());
+        }
+        let spt: proofs::RegisteredSealProof = aggregate.seal_proof.try_into()?;
+        let prover_id = prover_id_from_u64(aggregate.miner);
+
+        struct AggregationInputs {
+            commr: [u8; 32],
```

Comment:
readability: better to expand `r` and `d` to `replica` and `data` respectively?

Reply:
Yeah, or add comments

Reply:
commr and commd are colloquially used terms, but ill add comments.